app/store/chat.ts

```diff
@@ -282,7 +378,25 @@ export const useChatStore = createPersistStore(
         return session;
       },
 
-      onNewMessage(message: ChatMessage) {
+      async onNewMessage(message: ChatMessage) {
```

Comment:
Save new message of bot to supabase

Reply:
with this approach, we will have to query and insert to database for every message, which could mean a huge read/write small requests to database that is not desirable. How about, we have a frequent sync of local storage to database like every 1 minutes or when a session is started or ended, we update db if there are new messages in local storage since last sync? @duchuy31